release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3572,4 +3572,28 @@ For more information, see xref:../authentication/bound-service-account-tokens.ad
 [id="ocp-4-10-39-upgrading"]
 ==== Updating
 
-To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-40"]
+=== RHBA-2022:7298 - {product-title} 4.10.40 bug fix update
+
+Issued: 2022-11-09
+
+{product-title} release 4.10.40, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:7298[RHBA-2022:7298] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7297[RHBA-2022:7297] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.40 --pullspecs
+----
+
+[id="ocp-4-10-40-bug-fixes"]
+==== Bug fixes
+
+* Previously, `noAllowedAddressPairs` settings applied to all subnets on the same network instead of the one matching its corresponding subnet. Due to this, all ports had no address pairs. With this fix, `noAllowedAddressPairs` is now in its proper setting, ensuring each port is filtered to match its corresponding subnet. (link:https://issues.redhat.com/browse/OCPBUGS-1951)[*OCPBUGS-1951*])
```

Comment:
I suggest changing this to 'Before this update...' from 'Previously'. Although I can't find  the source information as a reference, this came up when running Vale for release notes.

Reply:
Done! Updates have been made. :)